superset-frontend/src/SqlLab/main.less

```diff
@@ -421,6 +421,7 @@ div.tablePopover {
       text-overflow: ellipsis;
       font-size: 16px;
       flex: 1;
+      width: 50px;
```

Comment:
I agree with what @michael-s-molina suggested above. Setting a fixed width does not look like the optimal solution here.